### DISCUSSION TITLE
Usage of GitHub logo on my personal website

### DISCUSSION THREAD

**Discussion by Tsatsch** [community/community#30309] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

My name Is Alex and I wanted to ask whether I am allowed to use the official Invertocat logo for my personal resume website in the skills section. I checked the website https://github.com/logos but I am still not sure whether I need to attribute this logo on my website or can just let it as it is? 

You can find a screenshot of the skills section part of my website to see where and how I plan to use the Github logo.

Many thanks
Alexander Semenov
<img width="1076" alt="Screenshot 2022-08-21 at 19 26 10" src="https://user-images.githubusercontent.com/63811548/185803439-d56419e9-6874-4c5a-b137-6cee808e9833.png">

**Comment by ichenglin** [comment, marked as answer]
I'm not a professional in law,
but I could tell you **don't** need to **cite/mention** Github to use their **[permitted icons](https://github.com/logos)**,
as long as you follow the **guidelines** they mentioned.

Restrictions **includes** but is **not limited** to the **modification** and **impersonation** of their 
copyrighted logo. There shouldn't be much to worry about as long as you **obey their terms**.

**Reply by Tsatsch** [reply]
I see, thank you very much for the quick answer. I think I am just thinking too complicated :).

**Reply by radar-blip** [reply]
Yeah, you're over thinking. You should be fine.
